tests/test_updater_ng.py

```diff
@@ -109,14 +110,72 @@ def tearDown(self):
     # Logs stdout and stderr from the sever subprocess.
     self.server_process_handler.flush_log()
 
+  def _create_consistent_target(self, delegator_file: str, targetname: str):
+    """Create consistent targets copies of their non-consistent counterparts
+    inside the repository directory.
+
+    Args:
+      delegator_file: Name of the delegator metadata file.
+      targetname: Name of the target file.
+
+    """
+
+    delegator_path = os.path.join(self.client_directory, delegator_file)
+    delegator = Metadata.from_file(delegator_path)
```

Comment:
You can use `sslib_util.get_file_details` to get the hashes, no need of delegator and `Metadata` object.

Reply:
The problem is that I don't know what hash algorithm to use when calling `sslib.utils.get_file_details()`.
The only one with that knowledge is the delegator.

Reply:
It is not a problem, since you have all the data.

Reply:
Yep, you are right that it's better to load the data into a dict instead of instantiating an object and doing all the things
around it.